### PR TITLE
test: cucumber forced sync

### PR DESCRIPTION
**Pull request by Cifko**
## Description
Add cucumber test that tests forced sync to single node.

## How Has This Been Tested?
npm test -- --name "Force sync many nodes agains one peer"

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I have squashed my commits into a single commit.

### CODE REVIEW

**Comment by Cifko** [comment]
The problem is that syncing 1000 nodes is not working. 999 works fine. Do we forbid sync over 1000 blocks (I don't think so). Haven't look at the error yet.
There are two options right now
1) set it to 999 blocks, create new ticket for the 1k sync and change it later to 1k
2) leave it at 1k and investigate now

File: integration_tests/features/Sync.feature [hunk 1]
```diff
@@ -167,3 +167,24 @@ Scenario: Pruned mode network only
     Then node PNODE2 is at height 20
     Given I have a pruned node PNODE3 connected to node PNODE1 with pruning horizon set to 5
     Then node PNODE3 is at height 20
+
+  Scenario Outline: Force sync many nodes agains one peer
```
Comment (delta1) [inline integration_tests/features/Sync.feature:174]:
```suggestion
  Scenario Outline: Force sync many nodes against one peer
```

**Review by delta1 — CHANGES_REQUESTED** [review]
Looking good, just a typo. Testing locally now.

**Review by delta1 — DISMISSED** [review]
Works well!

**Comment by philipr-za** [comment]
> The problem is that syncing 1000 nodes is not working. 999 works fine. Do we forbid sync over 1000 blocks (I don't think so). Haven't look at the error yet.
> There are two options right now
> 
>     1. set it to 999 blocks, create new ticket for the 1k sync and change it later to 1k
> 
>     2. leave it at 1k and investigate now

I believe this must be a bug related to the pruning horizon which is by default 1000 blocks?

**Review by stringhandler — APPROVED** [review]

**Pull request merged**